django_migration_linter/sql_analyser.py

```diff
@@ -63,29 +63,45 @@ def has_default(sql, **kwargs):
         'code': '',
         'fn': has_default,
         'err_msg': ''
+    }, {
+        'code': 'IGNORED_MIGRATION',
+        'fn': lambda sql, **kw: re.search('dml-ignore', sql),
```

Comment:
Two things:
* You duplicate the `dml-ignore` here 
* it seems that you do not need to match a regex here, it is

I think `'fn': lambda sql, **kw: sql == IGNORED_MIGRATION` should be enough (where IGNORED_MIGRATION is being imported)
If this creates a circular dependency, you might need to move the constant IGNORED_MIGRATION out of the `migration_linter.py` and move it into a `constants.py` file for example (and import that in `__init__.py` so that we can access the constant easily for imports)

Reply:
Duplication removed. Regex still needed.